### ISSUE TITLE
/usr/bin/env: node: No such file or directory

### ISSUE THREAD

**Issue by tmoore82** [AugurProject/augur#84]
Following the instructions at http://www.augur.net/blog/the-augur-alpha-is-now-available-to-download, I've tried to install the Augur client. But when I run `grunt broswerify:build`, I get the following error:

/usr/bin/env: node: No such file or directory

I receive no errors leading up to this point. 

I'm running unbuntu 15.04, 64-bit, Linux 3.19.0-21-generic.

**Comment by scottzer0** [comment]
it looks as if you haven't installed node for your platform.

if you're more interested in running the client than development at this point, you can skip building the client and just use http://client.augur.net

while this won't always be bleeding-edge, we do update it frequently with the latest build.

**Comment by tmoore82** [comment]
Thanks for the response! 

TL;DR
- I've got Node.js installed, but it doesn't appear to have registered in usr/bin/env (researching this now)
- client.augur.net fails, as well with "Failed to connect to Ethereum"

I've installed Node.js using `sudo apt-get install nodejs` and confirmed that I'm currently running v0.10.25. 

When I visit http://client.augur.net, I get the following message:

    Failed to connect to Ethereum

    Augur requires a local node of the Ethereum client running

    Visit the ethereum github wiki for help installing the latest client

    If geth is installed:
    geth --rpc --rpccorsdomain http://client.augur.net --shh --unlock primary

even though I've confirmed with geth that I'm running version 0.9.33 and am able to get a server up and running with the following (from http://www.augur.net/blog/the-augur-alpha-is-now-available-to-download):

    geth --rpc --rpccorsdomain "http://client.augur.net http://localhost:8080" --shh --networkid 1010101 --datadir "~/.augur" --protocolversion 59 --bootnodes "enode://587aa127c580e61a26a74ab101bb15d03e121a720401f77647d41045eae88709b01136e30aba56d1feddff757d4a333f68b9a749acd6852f20ba16ef6e19855a@[::]:30303 enode://4014c7fa323dafbb1ada241b74ce16099efde03f994728a55b9ff09a9a80664920045993978de85cb7f6c2ac7e9218694554433f586c1290a8b8faa186ce072c@97.125.111.149:30303 enode://e30a4cd1ec5ca4dbd19c40785d7d61363b44110cb20f27a21c5009ec20f68548b1d0705ba4483cb889cf835e0774db4df0f062054b412b0e17dae063c875939e@[::]:30303 enode://b8092b9c390c40463f152bfb5e16837b435255db0780594bc473811e31c05ea0994c4cb435f2ce93eef59786477d445a558b2d8519e6afb614918ae0924ff727@24.4.140.216:30303" --unlock primary console

So it appears that ethereum was installed successfully and is running.

**Comment by tmoore82** [comment]
Okay, found the Node.js issue. According to https://github.com/joyent/node/issues/3911, this is a naming "error" that can occur with package manager installs on Ubuntu. It can be solved with the following command:

    ln -s /usr/bin/nodejs /usr/bin/node

After this, `grunt browserify:build` succeeded.

I received the same error message `Failed to connect to Ethereum` when I went to http://localhost:8080/. This part I find confusing. The server was running (logging messages scrolling across the screen), but I thought I might need to restart it. So I hit `CTRL+C`, at which point I was prompted for a password. After entering my password, the page reloaded and I'm now up and running. Is this standard behavior? I didn't see anywhere in the documentation that I would need to hit `CTRL+C` to get prompted. Or am I starting up Ethereum the wrong way?

Thanks!

**Comment by tmoore82** [comment]
Nm. This is all just new to me. The log messages are just piling on top of the Password prompt and pushing it up the screen. So I wasn't seeing it.
